From f8882c500c1cb6b45d800840ea0b24a8cfd88331 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marek=20Marczykowski-G=C3=B3recki?=
 <marmarek@invisiblethingslab.com>
Date: Fri, 1 Dec 2023 12:46:34 +0100
Subject: [PATCH] qubes: Make vendor check case-insensitive

Lenovo uses "LENOVO" in DMI, but "Lenovo" in metainfo.xml
---
 contrib/qubes/src/qubes_fwupdmgr.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/contrib/qubes/src/qubes_fwupdmgr.py b/contrib/qubes/src/qubes_fwupdmgr.py
index ee73af2f4..09bbcdd4a 100755
--- a/contrib/qubes/src/qubes_fwupdmgr.py
+++ b/contrib/qubes/src/qubes_fwupdmgr.py
@@ -361,7 +361,7 @@ class QubesFwupdmgr(FwupdHeads, FwupdUpdate, FwupdReceiveUpdates):
         root = tree.getroot()
         vendor = root.findtext("developer_name")
         if vendor is not None:
-            if vendor not in dmi_info:
+            if vendor.lower() not in dmi_info.lower():
                 raise ValueError("Wrong firmware provider.")
         if not downgrade and pversion.parse(version) <= pversion.parse(
             self.dmi_version
-- 
2.41.0

